Editors/vscode/package.json

```diff
@@ -58,6 +58,11 @@
                     "default": "",
                     "description": "(optional) The path of the swift toolchain. By default, sourcekit-lsp uses the toolchain it is installed in."
                 },
+                "sourcekit-lsp.inlayHints.enabled": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Whether inlay type annotations should be rendered in the editor. Enabled by default."
```

Comment:
Since this shows up in the settings, I think a better wording would be 
```
Render inlay type annotations in the editor.
```

I also don’t think it matters much that it’s enabled by default since the user will see the checkbox ticked anyway.